src/datatypes/scenario/Field.cpp

```diff
@@ -27,6 +33,12 @@ namespace spy::scenario {
             if (fieldState != FieldStateEnum::SAFE) {
                 Field::safeIndex.reset();
             }
+
+            if (fieldState == FieldStateEnum::ROULETTE_TABLE) {
```

Comment:
in der funktionsdokumentation sollte stehen, dass dadurch die felder gecleared werden (kein erwartetes verhalten von einem setter)